packages/flutter_tools/test/commands.shard/hermetic/shell_completion_test.dart

```diff
@@ -23,6 +23,7 @@ void main() {
     setUp(() {
       Cache.disableLocking();
       mockStdio = MockStdio();
+      mockStdio.stdout.terminalColumns = 80;
```

Comment:
nit. you can also use cascade operator here.

Reply:
done!